### DISCUSSION TITLE
Disable cached resources usage

### DISCUSSION THREAD

**Discussion by giva01121** [concourse/concourse#8235] [Help & Support]
Hello,
I'm using the latest concourse version: 7.7.1
My issue is that concourse is using cached versions of the SWCE/metadata-resource. Once this happens I'm getting wrong metadata. For example my job name is volume_quota_check, but the metadata is displaying the name of another job. Is there a way to use a new resource on each build and not using a cached one?
![image](https://user-images.githubusercontent.com/58251102/161565932-6f1533d2-17e7-4607-8f8d-1dc955a90750.png)

**Comment by taylorsilva** [comment]
Took a look around but it doesn't look like there's a way to tell Concourse to not use a cached resource.

Though looking at https://github.com/swce/metadata-resource it seems like the way its check works should avoid this problem. Are you setting a `passed` constraint on the `get` step by any chance? https://concourse-ci.org/get-step.html#schema.get.passed
